src/transformers/modeling_mt5.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+# Copyright 2018 Mesh TensorFlow authors, T5 Authors and HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch mT5 model. """
+
+from .configuration_mt5 import MT5Config
+from .modeling_t5 import T5ForConditionalGeneration, T5Model
+from .utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+_CONFIG_FOR_DOC = "T5Config"
+_TOKENIZER_FOR_DOC = "T5Tokenizer"
+
+####################################################
+# This dict contains shortcut names and associated url
+# for the pretrained weights provided with the models
+####################################################
+T5_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "google/t5-small",
+    "google/t5-base",
+    "google/t5-large",
+    "google/t5-3b",
+    "google/t5-11b",
+    # See all T5 models at https://huggingface.co/models?filter=t5
+]
```

Comment:
Is this needed here? If it is, it probably shouldn't be those model identifiers but `mT5-{Small, Base, Large, XL, XXL}` instead